portfolio/src/main/java/com/google/sps/servlets/BlogCommentsServlet.java

```diff
@@ -14,19 +14,40 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.data.BlogComments;
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.Date;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+@WebServlet("/blog-comments")
+public final class BlogCommentsServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // Comments data
+    Date postDate = new Date();
+    String comment = "That's a wonderful blog!";
```

Comment:
Consider removing the temporary logic to another function so it's easy to see what this method will look like finally.

Reply:
Okay that's a great insight, I'll keep that in mind for my next PRs.